# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmBuiltins.h
# Commit: 25adf505426d
# Full Hash: 25adf505426d21f67e15d146f879b0ea53d343a7
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2022-03-23 04:09:13
# Regressor Bug: 1642412
# File Overlap Count: 19
# Description:
#   Bug 1642412 - wasm: Rollup of inline table.get/set for anyref. r=lth
#   
#   Rollup of the previous inline table.get/set patches.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D141024
# ==============================================================================

diff -r 43f7556db2e1 -r 25adf505426d js/src/wasm/WasmBuiltins.h
--- a/js/src/wasm/WasmBuiltins.h	Tue Mar 22 19:44:13 2022 +0000
+++ b/js/src/wasm/WasmBuiltins.h	Tue Mar 22 19:59:05 2022 +0000
@@ -112,11 +112,10 @@
   TableCopy,
   ElemDrop,
   TableFill,
-  TableGet,
+  TableGetFunc,
   TableGrow,
   TableInit,
-  TableSet,
-  TableSize,
+  TableSetFunc,
   RefFunc,
   RefTest,
   RttSub,
@@ -236,11 +235,10 @@
 extern const SymbolicAddressSignature SASigTableCopy;
 extern const SymbolicAddressSignature SASigElemDrop;
 extern const SymbolicAddressSignature SASigTableFill;
-extern const SymbolicAddressSignature SASigTableGet;
+extern const SymbolicAddressSignature SASigTableGetFunc;
 extern const SymbolicAddressSignature SASigTableGrow;
 extern const SymbolicAddressSignature SASigTableInit;
-extern const SymbolicAddressSignature SASigTableSet;
-extern const SymbolicAddressSignature SASigTableSize;
+extern const SymbolicAddressSignature SASigTableSetFunc;
 extern const SymbolicAddressSignature SASigRefFunc;
 extern const SymbolicAddressSignature SASigPreBarrierFiltering;
 extern const SymbolicAddressSignature SASigPostBarrier;